src/hf_hydrodata/data_catalog.py

```diff
@@ -467,28 +468,34 @@ def _get_preferred_catalog_entry(entries: List[dict]) -> dict:
         result = entries[0]
     else:
         preferred_file_types = ["pfb", "tif", "netcdf"]
```

Comment:
So it looks like we are preferring files in the order of pfb, tif, netcdf?  is that accurate?